evm-template/node/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = { workspace = true }
 license = { workspace = true }
 publish = false
 repository = { workspace = true }
-version = "0.1.2"
+version = "1.0.0"
```

Comment:
If we are doing a single release with the v2 tag does it make sense to keep separate versions for individual crates?